### PR TITLE
Merge upstream.

### PR DESCRIPTION
**Pull request by aak-code**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-  "version": "17.1.0",
-  "keywords": [
-    "Twitter",
-    "OAUTH",
-    "JavaScript",
-    "tweets",
-    "api",
-    "widget"
-  ],
+  "version": "18.0.2",
```
Comment (thelornenelson) [inline package.json:3]: 18.0.3

**Review by thelornenelson — APPROVED** [review]
Nice work! Just a discrepancy in the version numbers.

File: package.json (hunk 1 above)
Comment (aak-code) [reply package.json:3]: ahaha, looks like they get their own numbers wrong! This is all coming directly from their merge

Comment (thelornenelson) [reply package.json:3]: Derp... Well as long as it gets fixed for our own release/tag, whatever that may be!

**Pull request merged**